ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -1,22 +1,85 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type ServiceStatus string
+
+const (
+	FAIL_TO_GET_RAYSERVICE           ServiceStatus = "fail_to_get_rayservice"
```

Comment:
I believe the K8s uses UpperCamelCase convention for string enums, as in
`ImagePullPolicy: IfNotPresent`

Reply:
sgtm